cucumber/authenticators_azure/features/authn_azure_bad_configuration.feature

```diff
@@ -114,8 +114,8 @@ Feature: Azure Authenticator - Bad authenticator configuration leads to an error
     And I have host "test-app"
     And I set Azure annotations to host "test-app"
     And I grant group "conjur/authn-azure/prod/apps" to host "test-app"
-    And I successfully set Azure variables with the correct values
-    And I fetch an Azure access token from inside machine
+    And I successfully set Azure provider-uri variable with the correct values
```

Comment:
Do you think that in the future we will need other variables for authn-azure configuration besides `provider-uri`? 
If so, I would avoid being specific "I successfully set Azure provider-uri variable with the correct values" -> "I successfully set Azure "provider-uri" variable with the correct values" so that we can replace that with any `provider_uri` with any variable and it will functionally be the same

Reply:
I think this is a  better way, because here the user will understand which variable the automation set and it will be easy to debug in case of failure.
In my vision if you see the results via jenkins/report you will understand exactly the test and the parameters.

Reply:
i don't think the azure authenticator will have additional variables in the future. If it will have, it won't be hard to replace it. I agree with elad that the value of explicitly saying what the step does is high.